### ISSUE TITLE
MinGW CI is broken

### ISSUE THREAD

**Issue by vrabaud** [AOMediaCodec/libavif#1912]
`imagemagick` is the culprit and there are two reasons:
- on mingw32:
```
C:\Windows\system32\cmd.exe /D /S /C D:\a\_temp\setup-msys2\msys2.cmd -c "'pacboy' '--noconfirm' '-S' '--needed' 'aom:p' 'cmake:p' 'diffutils:p' 'cc:p' 'dav1d:p' 'imagemagick:p' 'libjpeg-turbo:p' 'libpng:p' 'libyuv:p' 'ninja:p' 'zlib:p'"
  error: target not found: mingw-w64-i686-imagemagick
  Error: The process 'C:\Windows\system32\cmd.exe' failed with exit code 1
```
- on the other platforms, the `test_cmd_icc_profile` test fails:
```
magick avif_test_cmd_icc_profile_decoded.png -profile D:/a/libavif/libavif/tests/data/sRGB2014.icc avif_test_cmd_icc_profile_corrected.png
magick.exe: no decode delegate for this image format `PNG' @ error/constitute.c/ReadImage/746.
```

**Comment by vrabaud** [comment]
@kmilos , any idea? Thx!

**Comment by kmilos** [comment]
MINGW32 is being obsoleted: https://www.msys2.org/news/#2023-12-13-starting-to-drop-some-32-bit-packages

**Comment by vrabaud** [comment]
Thx for the prompt response: I checked/tried whether there was a way to impose a version with pacboy on the CI but that seems not to be the case.

**Comment by kmilos** [comment]
> I checked/tried whether there was a way to impose a version with pacboy on the CI but that seems not to be the case.

Indeed, it's a rolling release like Arch, so you can't pin packages... You might be able to install old/archived versions manually from URL though, like `pacman -U https://repo.msys2.org/mingw/mingw32/mingw-w64-i686-imagemagick-7.1.1.23-1-any.pkg.tar.zst`

**Comment by vrabaud** [comment]
That has been fixed in https://github.com/msys2/MINGW-packages/commit/46ac18b5740f3c4622f8c48421f187aa229c8ec0
